style.css

```diff
@@ -1,80 +1,254 @@
+
 html {
   box-sizing: content-box;
-  display: flex;
-  justify-content: center;
-  background-color: rgba(100, 0, 0, 0.1);
+  -moz-box-sizing: content-box;
+  -webkit-box-sizing: content-box;
   font-size: 10px;
 }
 
 body {
-  background-color: #fff;
+  color: #3a4a42;
+  font-family: 'Inter', sans-serif;
+  font-weight: 600;
+  font-style: normal;
+}
+
+main {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 68px;
+  align-content: center;
+  align-items: center;
+}
+
+header {
+  text-align: center;
+}
+
+.projects {
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-content: center;
+}
+
+button {
+  border-style: none;
+}
+
+ul {
+  margin: 1px;
+  padding: 0.000001px;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+img {
   width: 100%;
-  border: 1px solid #999;
-  color: #172b4d;
-  justify-content: space-around;
-  flex-direction: column;
-  font-family: sans-serif;
-  font-family: 'Inter', serif;
+}
+
+h2 {
+  font-size: 4rem;
+  font-weight: 700;
+}
+
+.projects * {
+  margin: 0.000000001px;
+}
+
+h3 {
+  font-size: 2.4rem;
+  padding: 1.6rem;
+  font-weight: 600;
+  font-style: normal;
+}
+
+.medium {
   font-size: 2rem;
 }
 
-.tool-bar {
-  display: flex;
-  justify-content: space-between;
-  align-items: baseline;
-  height: 92px;
-  margin: 1px 24px;
+.medium-text {
+  font-size: 2rem;
+  font-weight: 400;
+  font-style: normal;
 }
 
-.background {
-  height: 726px;
-  background-image: url("./assets/header-illsutration-mobile@2x.png");
-  background-size: 100%;
+.about-me * {
+  margin: 1px;
+}
+
+.work * {
+  margin: 0.00000000000001px;
+}
+
+.work {
+  margin: 0.0000000000001px;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-auto-rows: 1fr;
+  grid-gap: 24px;
+  justify-content: center;
+}
+
+.about-me {
+  margin: 1px;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-auto-rows: 1fr;
+  grid-gap: 24px;
+  justify-content: center;
+}
+
+.hr {
+  width: 48px;
+  height: 4px;
+  margin: 24px 1px 68px;
+}
+
+.bg {
+  background: url('../assets/backgrounds/about-me-bottom.png') left bottom no-repeat, url("../assets/backgrounds/about-me-top.png") right top no-repeat;
+  background-size: 50px;
   background-repeat: no-repeat;
+  padding: 50px 1px 1px 1px;
+}
+
+.bg-light-green {
+  background-color: #ebf0ee;
+}
+
+.place-holder {
+  width: 100%;
+  height: 252px;
+}
+
+.bg-green {
+  background-color: #36b37f;
+}
```

Comment:
## [OPTIONAL]
- [ ] To improve your design I would suggest adding a hover effect on the green btn like this:
```
.button:hover {
    cursor: pointer;
}
```